routing/pathfind_test.go

```diff
@@ -3130,3 +3131,186 @@ func dbFindPath(graph *channeldb.ChannelGraph,
 
 	return route, err
 }
+
+// TestBlindedRouteConstruction tests creation of a blinded route with the
+// following topology:
+//
+// A -- B -- C (introduction point) - D (blinded) - E (blinded).
+func TestBlindedRouteConstruction(t *testing.T) {
```

Comment:
Hmm I find this testcase misses the whole fee/cltv calculation for the blinded paths, is this tested elsewhere, if so I would prefer a comment describing what's actually tested here, meaning that we are just nulling the values of the route, all the fee and cltv stuff for blinded routes is not tested here.